sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpConnectionScope.cs

```diff
@@ -72,6 +72,13 @@ internal class AmqpConnectionScope : TransportConnectionScope
         /// </summary>
         private static int AuthorizationBaseJitterSeconds { get; } = 30;
 
+        ///// <summary>
```

Comment:
I think there may have accidentally been more forward-slashes than you wanted

Reply:
Ugh.. This is from when I was testing something and had commented out the new property.

Reply:
https://github.com/Azure/azure-sdk-for-net/pull/30988